frontend/src/App.tsx

```diff
@@ -6,27 +6,55 @@ import DetailPage from "./pages/DetailPage.tsx";
 import EditPage from "./pages/EditPage.tsx";
 import {useFetch} from "./hooks/useFetch.ts";
 import {styled} from "styled-components";
+import HomePage from "./pages/HomePage.tsx";
+import {Button} from "@mui/material";
+import {ToastContainer} from "react-toastify";
+import 'react-toastify/dist/ReactToastify.css';
+
 
 export default function App() {
     const meetings = useFetch((state) => state.meetings);
 
     return (
         <main>
             <Header />
-
+            <ToastContainer/>
             <Routes>
-                <Route path="/" element={<MeetingList meetings={meetings} />} />
-                <Route path="/add" element={<AddPage />} />
+                <Route path="/" element={<HomePage/>}/>
+                <Route path="/meetinglist" element={<MeetingList meetings={meetings} />} />
+                <Route path="/addmeeting" element={<AddPage />} />
                 <Route path="/:id" element={<DetailPage />} />
-                <Route path="/:id/edit" element={<EditPage/>} />
+                <Route path="/:id/editmeeting" element={<EditPage/>} />
             </Routes>
 
             <StyledApp>
-                <Link to={"/"}>
-                    <button>My Meetings</button>
+                <Link to={"/meetinglist"}>
+                    <Button
```

Comment:
When the 2 buttons have the same styling consider adding a style component on the end of this file like this:

const StyledButton = styled(Button)`
  background-color: #d1adee;
  color: black;
  ....
`;

Reply:
yes I tried that but idk why but it didn't worked as I intended. thank you for your suggestion!